starlark/int.go

```diff
@@ -86,9 +87,20 @@ func (i Int) Int64() (_ int64, ok bool) {
 	return iSmall, true
 }
 
-// BigInt returns the value as a big.Int.
-// The returned variable must not be modified by the client.
+// BigInt returns a copy of the value as a big.Int.
```

Comment:
// BigInt returns a new big.Int with the same value as the Int.

(and strike the second line